sources/academy/webscraping/scraping_basics_python/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Web scraping basics for Python devs
+description: Learn how to use Python to extract information from websites in this practical course, starting from the absolute basics.
+sidebar_position: 10
+category: web scraping & automation
+slug: /scraping-basics-python
+---
+
+import DocCardList from '@theme/DocCardList';
+
+**Learn how to use Python to extract information from websites in this practical course, starting from the absolute basics.**
+
+---
+
+:::danger Work in progress
+
+This course is incomplete. As we work on adding new lessons, we would love to hear your feedback. Comment right here under each page or [file a GitHub Issue](https://github.com/apify/apify-docs/issues) to discuss a problem.
+
+:::
+
+In this course we'll use Python to create an application for watching prices. It'll be able to scrape all product pages of an e-commerce website and record prices. Data from several runs of such program would be useful for seeing trends in price changes, detecting discounts, etc.
+
+<!--
+TODO image of warehouse with some CVS or JSON exported, similar to sources/academy/webscraping/scraping_basics_javascript/images/beginners-data-collection.png, which is for some reason the same as sources/academy/webscraping/scraping_basics_javascript/images/beginners-data-extraction.png
+-->
+
+![E-commerce listing on the left, JSON with data on the right](./images/scraping.png)
+
+## What you'll do
+
+- Inspect pages using browser DevTools
+- Download web pages using the HTTPX library
+- Extract data from web pages using the Beautiful Soup library
+- Save extracted data in various formats, e.g. CSV which MS Excel or Google Sheets can open
+- Follow links programatically (crawling)
+- Save time and effort with frameworks, such as Scrapy, and scraping platforms, such as Apify
```

Comment:
Nitpick: About the lists of bullet points - I would use either a small letter at the beginning, with no punctuation at the end (or comma). Or a capital letter at the beginning and punctuation at the end.

Reply:
I have no idea how this is done correctly in English and whether there are any rules about it, so I just freestyle my way through it. I know in Czech there's pretty exact rule about capital letters and punctuation in lists like this, and in Czech text I tend to follow it. I'll search the internet and if there's such a rule, I'll fix all bullet point lists to be consistent with it. Maybe @TC-MO can also advise.

Reply:
I did [some reading](https://proofed.com/writing-tips/how-to-punctuate-and-capitalize-bullet-points/) and it seems the rules are quite loose. I opt for having full sentences with a capital letter and a punctuation at the end, and fragments of sentence to start with lower case and end with something like comma (which is the Czech rule I'm basically following elsewhere in the course).

Reply:
Addressed in 531ca173

Reply:
oh somehow this ping eluded me... either way, both are fine though one thing I'll mention whenever such lists are composed to try and remember to follow Parallel Structure as well.

Reply:
First time I hear about Parallel Structure, I'll check it out, thanks!